Source/SwiftLintFramework/Rules/SwitchCaseAlignmentRule.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  SwitchCaseAlignmentRule.swift
+//  SwiftLint
+//
+//  Created by Austin Lu on 9/6/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+public struct SwitchCaseAlignmentRule: ASTRule, OptInRule, ConfigurationProviderRule {
```

Comment:
Why is this marked as opt-in? From what I can tell, the community has (mostly) settled on `case` being vertically aligned with `switch` following Xcode's default formatting.

Reply:
I agree this rule shouldn't be controversial, and I'm happy to make it enabled by default. PR updated!